ihatemoney/templates/forms.html

```diff
@@ -93,8 +93,11 @@
     {{ form.hidden_tag() }}
     {{ input(form.date, class="form-control", inline=True) }}
     {{ input(form.what, inline=True) }}
+    {% if form.is_loan() %}
```

Comment:
with the change, it would be `if form.type == "loan"`

Reply:
Personally I prefer use a function that contains the check logic but if you think it's a better way to handle this I can change it.
Why do I prefer a function ? What if we change the string in the future ? If you use a function you only change BillTypes.loan value one time, if not you have to check where all `"loan"` occurrences are ...

Reply:
It depends. Here the check is so simple that I greatly prefer to check using strings: it's less indirection, and if we ever change the string in the future, it'll still be really easy to change.

« Premature optimization is the root of all evil ».